ladon.go

```diff
@@ -24,11 +24,19 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Metric is used to expose metrics about authz
+type Metric interface {
+	RequestDeniedBy(Request, Policy)
```

Comment:
Please add some documentation here :) E.g.

```
// RequestDeniedBy is called when ...
```

Would be great if you could do that for the other methods as well - you can also remove my comment `// <- no match also means no policy!`